linter.py

```diff
@@ -1,5 +1,21 @@
+import os
 from SublimeLinter.lint import NodeLinter
 
+# TODO Proper export these in SL core
+from SublimeLinter.lint.linter import PermanentError
+from SublimeLinter.lint.base_linter.node_linter import read_json_file
+
+
+STANDARD_SELECTOR = 'source.js'
+PLUGINS = {
+	'eslint-plugin-svelte3': 'text.html',
+	'eslint-plugin-vue': 'text.html.vue',
+	'eslint-plugin-json': 'source.json',
+	'eslint-plugin-html': 'text.html'
+}
```

Comment:
Um, obviously this is a mapping you need to adjust to your liking.

Reply:
Can you add https://github.com/eslint/eslint-plugin-markdown ?

Reply:
Done